src/index.js

```diff
@@ -29,7 +29,7 @@ function macros(babel) {
                 envFlags: { envFlagsImport, flags }
               } = options;
 
-              let isFeaturesImport = featureSources.includes(importPath);
+              let isFeaturesImport = featureSources.indexOf(importPath) > -1;
```

Comment:
😢 but it seems easier to use `indexOf` than add a dep on `core-js` for the polyfill...